src/Components/Publishing/Sections/__tests__/Text.test.tsx

```diff
@@ -8,72 +9,102 @@ import {
   TextFeatureArticle,
   TextStandardArticle,
 } from "../../Fixtures/Articles"
+import { ContextProvider } from "../../../Artsy"
```

Comment:
Since @l2succes got absolute paths working we should start the process of incrementally migrating imports

Reply:
Good to know, didn't realize we had this -- @l2succes have you posted this to any dev channels?

Reply:
agreed. @eessex Yes I did, I made an announce in the #dev channel